src/utils/types.js

```diff
@@ -7,9 +7,11 @@ export default {
     text: '#d47b00'
   },
   casts: {
-    'any': ['image', 'number', 'boolean', 'text'],
-    'boolean': ['number', 'text'],
-    'text': ['number']
+    'any': ['any', 'image', 'number', 'boolean', 'text'],
+    'image': ['any'],
+    'number': ['any'],
+    'boolean': ['any', 'number', 'text'],
+    'text': ['any', 'number']
```

Comment:
This enables casting *from* `any` to all other types. I have done this in preparation for the Group Input node, which uses this (see `groups` branch). However, we need to be somewhat careful with this now, since outputting `any` may now lead to unexpected results. (In the case of groups, the output is immediately transformed to the connected type.)